app/cdap/components/GridTable/components/Breadcrumb/index.tsx

```diff
@@ -21,25 +21,42 @@ import { useStyles } from './styles';
 import React from 'react';
 import { getCurrentNamespace } from 'services/NamespaceStore';
 import { Link } from 'react-router-dom';
+import T from 'i18n-react';
 
-export default function BreadCrumb({ datasetName }) {
+export default function Breadcrumb({ workspaceName, location }) {
   const classes = useStyles();
+
+  const sourcePath =
+    location?.state?.from === T.translate('features.WranglerNewUI.Breadcrumb.labels.wrangleHome')
```

Comment:
you could do 
```
const textPrefix = 'features.WranglerNewUI.Breadcrumb';
T.translate(`${textPrefix}.labels.wrangleHome`);
```
and save having to write out the whole string every time?

Reply:
Done